src/class/IContentTypes/IContentTypes.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef I_CONTENT_TYPES_HPP
+#define I_CONTENT_TYPES_HPP
+
+#include <iostream>
+#include <list>
+#include <map>
+#include <string>
+
+#ifdef TEST_ON
+#define private public
+#define protected public
+#endif
+
+class IContentTypes
```

Comment:
why is this an interface?
usually interfaces are "agreements" between classes.
also, interfaces should be only functions with no vars.

Reply:
you can also change it to a namespace, but im not sure what is the best practice haha

Reply:
I change it for the next PR